fakecore/src/any.rs

```diff
@@ -48,6 +49,7 @@ impl<'out> Request<'out> {
             );
```

Comment:
but this line 

```rust
            let this = unsafe { &mut *(self as *mut _ as *mut RequestBuf<Option<&'out T>>) };
```

probably needs to be checked to `self.get_unchecked_mut() as *mut _`